src/ServicePulse.Host/vue/src/assets/navbar.css

```diff
@@ -1,241 +1,267 @@
-    font-weight: normal;
-    text-decoration: none;
+  font-weight: normal;
+  text-decoration: none;
 }
 
 .navbar a:hover {
-    font-weight: normal;
+  font-weight: normal;
 }
 
 .navbar-inverse .navbar-nav > .active > a,
 .navbar-inverse .navbar-nav > .active > a:hover,
 .navbar-inverse .navbar-nav > .active > a:focus {
-    color: #fff;
+  color: #fff;
 }
 
 .label-important,
 .badge-important {
-    background-color: #FA603D;
-    border-color: #FA5833;
+  background-color: #fa603d;
+  border-color: #fa5833;
+}
+
+a.btn-feedback,
+a.btn-feedback:visited,
+a.btn-feedback:focus {
+  color: #fff !important;
+  background-color: #00a3c4 !important;
+  padding: 6px 16px !important;
+  margin: 14px 0 0 0;
+  border-radius: 3px;
+  font-weight: bold;
 }
 
-a.btn-feedback, a.btn-feedback:visited, a.btn-feedback:focus {
-	color: #fff !important;
-	background-color: #00A3C4 !important;
-	padding: 6px 16px !important;
-	margin: 14px 0 0 0;
-	border-radius: 3px;
-	font-weight: bold;
+a.btn-feedback:hover,
+a.btn-feedback:focus {
+  background-color: hsl(194, 95%, 61%) !important;
+  font-weight: bold;
 }
 
 .navbar-nav > li > .btn-feedback > .fa {
-    color: #fff;
+  color: #fff;
 }
 
 @media (min-width: 768px) {
-    .navbar-nav > li.active > a {
-        background: transparent !important;
-        border-bottom: 5px solid #00A3C4;
-    }
-
-    .navbar-nav > li > a {
-        padding-bottom: 15px;
-        padding-top: 20px;
-    }
-
-    .graph-values .col-sm-6 {
-        width: 45%;
-    }
+  .navbar-nav > li.active > a {
+    background: transparent !important;
+    border-bottom: 5px solid #00a3c4;
+  }
+
+  .navbar-nav > li > a {
+    padding-bottom: 15px;
+    padding-top: 20px;
+  }
+
+  .graph-values .col-sm-6 {
+    width: 45%;
+  }
 }
 
 @media only screen and (min-width: 1072px) {
-    .navbar-label {
-        display: inline;
-    }
+  .navbar-label {
+    display: inline;
+  }
 }
 
 @media only screen and (max-width: 768px) {
-    .navbar-collapse.collapse.in {
-        padding: 0 0 0 16px !important;
-    }
+  .navbar-collapse.collapse.in {
+    padding: 0 0 0 16px !important;
+  }
 }
 
 @media (max-width: 1199px) {
-    .navbar-header {
-        float: none;
-    }
+  .navbar-header {
+    float: none;
+  }
+
+  .navbar-toggle {
+    display: block;
+  }
 
-    .navbar-toggle {
-        display: block;
-    }
+  .navbar-collapse {
+    border-top: 1px solid transparent;
+    box-shadow: inset 0 1px 0 rgba(255, 255, 255, 0.1);
+  }
 
-    .navbar-collapse {
-        border-top: 1px solid transparent;
-        box-shadow: inset 0 1px 0 rgba(255,255,255,0.1);
-    }
+  .navbar-collapse.collapse {
+    display: none !important;
+  }
 
-    .navbar-collapse.collapse {
-        display: none !important;
-    }
+  .navbar-nav {
+    float: none !important;
+    margin: 7.5px -15px;
+  }
 
-    .navbar-nav {
-        float: none !important;
-        margin: 7.5px -15px;
-    }
+  .navbar-nav > li {
+    float: none;
+    float: none;
```

Comment:
Seems to be duplicated?

Reply:
yep: wasn't changed in this PR other than whitespace, but will fix anyway